migrations/004.do.indexer-stats.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE indexer_query_stats (
+  day DATE NOT NULL PRIMARY KEY,
+  total INT NOT NULL,
+  advertising_http INT NOT NULL
+);
```

Comment:
Do we want to keep track of the number of deals advertising Graphsync? On the one hand, it tells us how many SPs are advertising to IPNI (which is good) but not supporting HTTP yet (which may be easy to fix). OTOH, we don't really care about this metric, our ultimate goal is to have everybody advertising HTTP.

@patrickwoodhead thoughts?